src/components/ServiceRequestForm.jsx

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from "react";
+import { Formik, Form, Field } from "formik";
+import Categories from "./services/categories.json";
+import * as Yup from "yup";
+import ErrorMsg from "./errormsg";
+import FormContainer from './FormContainer';
+
+const getSubCategories = (categories, categoryId) => {
+	var category = categories.find(category => category.id === categoryId);
+	return category ? category.types : [];
+};
+
+const ServiceRequestForm = props => {
+
+	const [categories] = useState(Categories);
+	const [subCategories, setSubCategories] = useState([]);
+
+	const handleServiceRequestChange = (changeEvent, setFieldValue) => {
+		const { value } = changeEvent.currentTarget;
+		const subCategories = getSubCategories(categories, parseInt(value));
+		setFieldValue(changeEvent.currentTarget.name, changeEvent.currentTarget.value)
+		setFieldValue('subRequestType', '')
+		setSubCategories(subCategories);
+	};
+
+
+	return (
+
+		<React.Fragment>
+			<FormContainer title="How Can We Help?">
+				<Formik
+					initialValues={{
+						requestType: '',
+						subRequestType: '',
+					}}
+					validationSchema={Yup.object().shape({
+						requestType: Yup.string().required('Request Category is required'),
+						subRequestType: Yup.string().required('Sub Category is required'),
+					})}
+
+					onSubmit={(values, { setSubmitting }) => {
+
+						alert(JSON.stringify(values, null, 2));
+						setSubmitting(false);
+
+					}}
+				>
+					{
+						(props) => {
+							const { values, isSubmitting, errors, touched } = props;
+
+							return (
+
+								<Form >
+									<label htmlFor="requestType"
+										className={
+											errors.requestType && touched.requestType ? "input-feedback" : "text-label"}
+
+									>Request Category</label>
+									<br />
```

Comment:
please remove any `<br>` tags, this will be handled with css. I think somehow this line got moved, we dont' want to remove the submit buttons.

Reply:
<br>'s looked removed